R_program/CallSTEPWAT2.R

```diff
@@ -198,6 +278,13 @@ foreach (g = 1:length(GCM)) %dopar% { # loop through all the GCMs
               system(paste0("mv ", "sxwprod_v2.", downscaling_method, ".", y, ".", r, ".", GCM[g], ".in", " sxwprod_v2.in"))
             }
             
+            # If the user didn't specify this particular GCM/RCP combination
+            if(!dir.exists(weather.dir)) {
+              next;
+            } else {
+              setwd(weather.dir)
```

Comment:
I propose to change this to weather.read.dir, so it does not get confused with weather.dir below.